.eslintignore

```diff
@@ -1,17 +1,22 @@
 build/**
 components/**
+coverage/**
 dist/**
 libtextsecure/**
-coverage/**
 
 # these aren't ready yet, pulling files in one-by-one
-js/**
-test/**
+js/*.js
+js/models/**/*.js
+js/react/**/*.js
+js/views/**/*.js
+test/*.js
+test/models/*.js
+test/views/*.js
 /*.js
+
+# ES2015+ files
+!js/background.js
+!js/models/conversations.js
+!js/views/file_input_view.js
 !main.js
 !prepare_build.js
-
-# all of these files will be new
-!test/server/**/*.js
-
-# all of app/ is included
```

Comment:
I debugged this quite a bit and empirically found that `!` negation rule only works for me when not using the unqualified `**` glob pattern above, see: https://github.com/signalapp/Signal-Desktop/commit/5c31291a1a147f8874f6794d1032bf5507434bcb

Luckily, we only have a few folders, so excluding them explicitly is not a huge deal. The upside of this is that new files will most likely fall under ESLint process. Tested using `$(npm bin)/eslint .` to confirm it didn’t trigger `--no-ignore` warnings.